sdk/lib/atl/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 
 add_library(atl_classes INTERFACE)
 if(DBG)
-    target_compile_definitions(atl_classes INTERFACE _DEBUG)
+    #target_compile_definitions(atl_classes INTERFACE _DEBUG) # HACK & FIXME: CORE-17505
```

Comment:
Does this need to be disabled globally, or could it be done on the affected files/modules only?

Reply:
If you have any other idea, please create your Pull Request.

Reply:
Pinpoint fix is preferred but we can't.